sdk/webpubsub/azure-messaging-webpubsubservice/README.md

```diff
@@ -107,10 +133,32 @@ This SDK uses Python standard logging library.
 You can configure logging print out debugging information to the stdout or anywhere you want.
 
 ```python
+import sys
 import logging
+from azure.identity import DefaultAzureCredential
+>>> from azure.messaging.webpubsubservice import WebPubSubServiceClient
+
+# Create a logger for the 'azure' SDK
+logger = logging.getLogger('azure')
+logger.setLevel(logging.DEBUG)
+
+# Configure a console output
+handler = logging.StreamHandler(stream=sys.stdout)
+logger.addHandler(handler)
 
-logging.basicConfig(level=logging.DEBUG)
-````
+endpoint = "<endpoint>"
+credential = DefaultAzureCredential()
+
+# This client will log detailed information about its HTTP sessions, at DEBUG level
+client = WebPubSubServiceClient(endpoint=endpoint, credential=credential, logging_enable=True)
+```
+
+Similarly, `logging_enable` can enable detailed logging for a single call,
+even when it isn't enabled for the client:
+
+```python
+result = client.web_pub_sub.send_to_all(logging_enable=True)
```

Comment:
question: is `hub` optional for `send_to_all`?

Reply:
It is my carelessness. I just want to show how to open log tracing for single function.